### PR TITLE
Feature loader page spinner

### PR DESCRIPTION
**Pull request by Akinwaleadeola**

### CODE REVIEW

File: .vscode/settings.json
Comment (dabirideji) [inline .vscode/settings.json:1]: ???

File: WEB_ROOT/HTML/ADMIN/DASHBOARD/admin-dashboard.html
Comment (dabirideji) [inline WEB_ROOT/HTML/ADMIN/DASHBOARD/admin-dashboard.html:1]: Why working on this file .? PR does not require you to work on this file

File: WEB_ROOT/HTML/ADMIN/DASHBOARD/admin-dashboard.html [hunk 1]
```diff
@@ -263,12 +265,8 @@ <h2 class="text-lg font-medium text-[#34495e] flex items-center mb-4">
...
                                 <p>Referral Link:</p>
                                 </p>
-                                <a href="#" class="text-gray-600 italics">
-                                    https://metrohyp.com/link/dabirideji
-                                </a><br>
-                                <button
-                                    class="bg-[#26B99A] opacity-80 border border-[#169F85] text-white text-sm py-2 px-4 rounded-lg mt-2">Copy
-                                    Link</button>
+                                <a href="#" id="text-to-copy" class="text-gray-600 italics"> https://metrohyp.com/link/dabirideji</a><br>
+                                <button  data-clipboard-target="#text-to-copy" class="bg-[#26B99A] opacity-80 border border-[#169F85] text-white text-sm py-2 px-4 rounded-lg mt-2 copy-btn">Copy Link</button>
```
Comment (dabirideji) [inline WEB_ROOT/HTML/ADMIN/DASHBOARD/admin-dashboard.html:269]: Why working on clipboard and loader spinner on d same pull request .

File: WEB_ROOT/HTML/LOADER SPINNER/LoaderSpinner.html
Comment (dabirideji) [inline WEB_ROOT/HTML/LOADER SPINNER/LoaderSpinner.html:1]: Bad file name . Change to HTML/SHARED/LOADER

Comment (dabirideji) [reply WEB_ROOT/HTML/LOADER SPINNER/LoaderSpinner.html:1]: Loader should not be an html but rather should be a JavaScript function

File: WEB_ROOT/JS/ADMIN/DASHBOARD/admin-dahboard.js
Comment (dabirideji) [inline WEB_ROOT/JS/ADMIN/DASHBOARD/admin-dahboard.js:1]: Wong feature .

File: WEB_ROOT/JS/LOADER SPINNER/LoaderSpinner.js
Comment (dabirideji) [inline WEB_ROOT/JS/LOADER SPINNER/LoaderSpinner.js:1]: Bad file name

File: WEB_ROOT/PLUGINS/IGNORED-CSS/LOADER SPINNER/LoaderSpinner.css
Comment (dabirideji) [inline WEB_ROOT/PLUGINS/IGNORED-CSS/LOADER SPINNER/LoaderSpinner.css:1]: Do not use whitespaces when creating file names

**Review by dabirideji — CHANGES_REQUESTED** [review]

File: .vscode/settings.json
Comment (Akinwaleadeola) [reply .vscode/settings.json:1]: i dont know anything about this.

File: WEB_ROOT/HTML/ADMIN/DASHBOARD/admin-dashboard.html
Comment (Akinwaleadeola) [reply WEB_ROOT/HTML/ADMIN/DASHBOARD/admin-dashboard.html:1]: For the copy to clipboard functionality

**Pull request merged**